core/src/main/java/org/apache/druid/java/util/common/parsers/DelimitedParser.java

```diff
@@ -74,6 +77,24 @@ public DelimitedParser(
   @Override
   protected List<String> parseLine(String input)
   {
-    return splitter.splitToList(input);
+    return splitToList(input);
+  }
+
+  /**
+   * Copied from Guava's {@link Splitter#splitToList(CharSequence)}.
+   * This is to avoid the missing method signature error when using an old Guava library.
```

Comment:
Could you write some more comments, or link to an issue, describing what situations this helps in?

Something like: this may happen when running Druid Hadoop indexing jobs, since we may inherit the version provided by the Hadoop cluster.

Reply:
Thanks. Added.